src/app/components/tree/tree.component.css

```diff
@@ -9,68 +9,75 @@
   Copyright Contributors to the Zowe Project.
 */
 
+
+
+/* !important tags are used here to prevent css bleeding from other FE instances */
```

Comment:
"!important tags are used here to prevent other apps to bleed into the FE style"

is more accurate